src/App.tsx

```diff
@@ -16,80 +18,130 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  NONE = 'NONE',
+  ALPHABET = 'ALPHABET',
+  LENGTH = 'LENGTH',
 }
 
 type ReorderOptions = {
   sortType: SortType,
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((prevGood, currentGood) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return prevGood.localeCompare(currentGood);
+      case SortType.LENGTH:
+        return prevGood.length - currentGood.length;
+      default:
+        return 0;
+    }
+  });
+
```

Comment:
Using ternary operator for return statement can make the code more readable. 😺 Change the if statement to ```return isReversed
  ? visibleGoods.reverse()
  : visibleGoods;
```